lib/msal-common/src/request/CancellationToken.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+* Cancellation token used for cancelling the device code request. 
+* 
+* While the user authenticates on a separate device, MSAL polls the the token endpoint of security token service for the interval 
+* specified in the device code resonse (usually 15 minutes). To stop polling and cancel the request, set cancellationToken.cancel = true. 
+* */
+export class CancellationToken {
```

Comment:
I may just be ignorant here, but is the plan to add to this class? what's the purpose of having a class with just one field and nothing else?

Reply:
CancellationToken is a pattern used to be able to short-circuit a long running operation.